styles/globals.css

```diff
@@ -22,23 +22,64 @@ h5,
 h6 {
   @apply tracking-tight;
 }
-button:not([role]){
+button:not([role]) {
   @apply border;
   @apply border-transparent;
 }
 
-input[type="checkbox"]:focus-visible;
-[role="checkbox"]:focus-visible;
-[role="radio"]:focus-visible;
-[role="combobox"]:focus-visible;
-[role="switch"]:focus-visible;
+input[type="checkbox"]:focus-visible,
+[role="checkbox"]:focus-visible,
+[role="radio"]:focus-visible,
+[role="combobox"]:focus-visible,
+[role="switch"]:focus-visible,
 button:not([role]):focus-visible {
   @apply outline-none;
   @apply ring;
   @apply border-orange-500;
   @apply ring-orange-100;
 }
 
+.transition-colors {
+  transition-property: color, background-color, border-color, text-decoration-color, fill, stroke;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 0.15s;
+}
+
+.transition-all {
+  transition-property: all;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 0.15s;
+}
+
+/* collapsible component animation */
+.CollapsibleContent {
+  overflow: hidden;
+}
```

Comment:
refactor (blocking): use the Tailwind CSS class, `overflow-hidden` for this instead.